torch/_dynamo/symbolic_convert.py

```diff
@@ -757,7 +766,12 @@ def wrapper(self: "InstructionTranslatorBase", inst: Instruction):
                     # We don't support graph break under GenericContextWrappingVariable,
                     # If there is, we roll back to the checkpoint and fall back.
                     excp.remove_from_stats()
-                    unimplemented("Graph break under GenericContextWrappingVariable")
+                    unimplemented_v2(
+                        gb_type="Graph break under GenericContextWrappingVariable",
+                        context="",
+                        explanation="Attempted to graph break in an active context manager that doesn't support graph breaking.",
```

Comment:
This is more that... "when there is a graph break in code under a generic context manager, we will graph break on the generic context manager".

Also, it would be nice if the error stack trace points to the graph break instead of the context manager

Reply:
Will be addressed in PR 3/N